Other/Badges/NoEgrets.cs

```diff
@@ -0,0 +1,29 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+//cs_include Scripts/Seasonal/HarvestDay/CoreHarvestDay.cs
+using Skua.Core.Interfaces;
```

Comment:
add this file to it's respective seasonal folder harvest day and rename file folder NoEgretsBadge

Reply:
Done 👍

Reply:
resolve my comment next time gonna merge also idk who u are in discord pls mention me